packages/@dcl/playground-assets/etc/playground-assets.api.md

```diff
@@ -503,6 +503,7 @@ export const componentDefinitionByName: {
     "core::UiInputResult": LwwComponentGetter<LastWriteWinElementSetComponentDefinition<PBUiInputResult>>;
     "core::UiText": LwwComponentGetter<LastWriteWinElementSetComponentDefinition<PBUiText>>;
     "core::UiTransform": LwwComponentGetter<LastWriteWinElementSetComponentDefinition<PBUiTransform>>;
+    "core::VideoEvent": LwwComponentGetter<LastWriteWinElementSetComponentDefinition<PBVideoEvent>>;

```

Comment:
Pravs, I think we may create this component as grow only set. Did you think about that?

Reply:
The use case is to also control the play/pause from the evets themselves

Reply:
Uhmm, you are right I'll change it to a GrowOnlySet since the event is only to read data after I finish testing some stuff using the LWW, thx!